src/types/Nutritionist.ts

```diff
@@ -0,0 +1,19 @@
+export type Nutritionist = {
+    id: number;
+    attributes: {
+        blocked: boolean;
+        confirmed: boolean;
+        createdAt: string;
+        email: string;
+        provider: string;
+        type: string;
+        updatedAt: string;
+        username: string;
+        metabolicRate: string;
+        age: number;
+        gender: string;
+        height: string;
+        weight: string;
+        activity: string;
+    };
+};
```

Comment:
This seems to be the same as the Client type, can you check if they are indeed the same and remove this if they are?